extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -566,6 +566,7 @@ export class DashboardWidget {
 		const refreshButton = view.modelBuilder.hyperlink().withProps({
 			label: loc.REFRESH,
 			url: '',
+			isButton: true,
```

Comment:
I believe there is already a property called ariaRole, you should leverage that instead of introducing a new property.